.github/workflows/web.yml

```diff
@@ -100,11 +100,12 @@ jobs:
       - uses: ./.github/actions/frontend-build
         with:
           base_directory: ${{ env.base_directory }}
+          dependencies_lock_file: ${{ env.base_directory }}/pnpm-lock.yaml
           index_file: ${{ env.base_directory }}/www/index.html
           static_directory: ${{ env.base_directory }}/www/static
           index_cache_key: ${{ github.sha }}-${{ github.run_id }}-index
           static_cache_key: ${{ github.sha }}-${{ github.run_id }}-static
-          command: centreon:build
+          build_command: pnpm centreon:build
```

Comment:
shouldn't you quote the command using ":" ?

Reply:
not needed :)